### DISCUSSION TITLE
Github Markdown Maths: Sum of as part of a fraction renders incorrectly

### DISCUSSION THREAD

**Discussion by gibbardsteve** [community/community#110637] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I am trying to render an equation that requires sum notation to be used as part of a fraction.

I expect the rendering to look like this:
![image](https://github.com/community/community/assets/88480716/3f14c08f-e762-4b53-ae63-9738fe16bb3b)

However when using the following expression:
```
$$\begin{equation}
\frac{\sum_{i=1}^{n_h} y_i}{n_h}
\end{equation}$$
```

It renders like this:
![image](https://github.com/community/community/assets/88480716/bef5a126-98ab-4a46-aaa2-d0022f02f762)

If I use the expression outside of a fraction the sum notation looks fine:
```
$$\begin{equation}
\sum_{i=1}^{n_j} a_i y_i
\end{equation}$$
```
![image](https://github.com/community/community/assets/88480716/afc3fd70-4a7f-40bf-932d-fc1a8d232103)

Any ideas or workarounds?

**Comment by gil-son** [comment]
Hello,

Try it:

```
$$\begin{equation}
\frac{\sum_{^{n_h} {i=1} y_i}}{n_h}
\end{equation}$$
```

The result is:

$$\begin{equation}
\frac{\sum_{^{n_h} {i=1} y_i}}{n_h}
\end{equation}$$


If I supported you consider to make my answer. God bless you!

**Comment by denass03** [comment]
I think you can use the \limits notation version of this instead like this:

```
$$\begin{equation}
\frac{\sum\limits_{i=1}^{n_h} y_i}{n_h}
\end{equation}$$
```

$$\begin{equation}
\frac{\sum\limits_{i=1}^{n_h} y_i}{n_h}
\end{equation}$$

**Comment by LikhithRanga** [comment]
You can replace that line with this

\frac{\sum_{i=1}^{n_h} y_i}{n_h}

**Comment by Buddhi19** [comment, marked as answer]
Well you can get exactly as shown in the figure!! using,
```$\frac{{\sum}^{n_h}_{i=1} y_i}{n_h}$```

$\frac{{\sum}^{n_h}_{i=1} y_i}{n_h}$
